src/LinuxTracing/LibunwindstackUnwinder.cpp

```diff
@@ -117,6 +126,92 @@ LibunwindstackResult LibunwindstackUnwinderImpl::Unwind(
   return LibunwindstackResult{unwinder.ConsumeFrames(), unwinder.LastErrorCode()};
 }
 
+std::optional<bool> HasFramePointerSet(
```

Comment:
`static`.

Reply:
I would also suggest renaming as this is not conceptually an overload of `LibunwindstackUnwinderImpl::HasFramePointerSet`, rather a utility function (e.g. `HasFramePointerSetFromDwarfSection`.

Reply:
> static 
All of this is in an unnamed namespace already so nothing is visible from the outside.

The function deserves a comment even if it is not visible by external users.

Reply:
Done.